we don't have code currently that uses the CLI tools, and the CLI tools require
the apputils project, and that doesn't currently work with python3.  drop it for
now to get general python3 support.
https://github.com/google/apitools/issues/8

--- a/setup.py
+++ b/setup.py
@@ -69,13 +69,7 @@ setuptools.setup(
     author_email='craigcitro@google.com',
     # Contained modules and scripts.
     packages=setuptools.find_packages(),
-    entry_points={'console_scripts': CONSOLE_SCRIPTS},
     install_requires=REQUIRED_PACKAGES,
-    tests_require=REQUIRED_PACKAGES + CLI_PACKAGES + TESTING_PACKAGES,
-    extras_require={
-        'cli': CLI_PACKAGES,
-        'testing': TESTING_PACKAGES,
-        },
     # Add in any packaged data.
     include_package_data=True,
     package_data={
